aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -246,6 +247,22 @@ module aptos_framework::genesis {
                 employee_group.validator.commission_percentage,
                 x"",
             );
+            let pool_address = vesting::stake_pool_address(contract_address);
+
+            let validator = &employee_group.validator.validator_config;
+            assert!(
```

Comment:
nit: You can use aptos_account::assert_account_exists: https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/aptos_account.move#L32

Reply:
let's see if @gregnazario or someone else reviews this first... I want to get out of your way